nncf/experimental/torch/nas/bootstrapNAS/elasticity/elastic_depth.py

```diff
@@ -93,6 +99,27 @@ def depth_indicator(self, depth_indicator: int) -> None:
         self._depth_indicator = depth_indicator
         self._is_search_space_obsolete = True
 
+    def load_state(self, state: Dict[str, Any]) -> None:
+        """
+        Initializes object from the state.
+        :param state: Output of `get_state()` method.
+        """
+        active_config = state[self._state_names.ACTIVE_CONFIG]
+        self.activate_subnet_for_config(active_config)
```

Comment:
SingleElasticityHandler adds `active_config`. Just reuse it in load_state and get_state of descendant classes.
```
super().load_state(state)
```

Reply:
Thanks! Done.